app/process/watcher/consensus-message/consensus-message.go

```diff
@@ -88,15 +90,29 @@ func (ctw ConsensusTopicWatcher) subscribeToTopic(q *queue.Queue) {
 	log.Infof("Started Consensus Message Watcher for topic [%s]\n", ctw.topicID)
 	unprocessedMessages, err := ctw.client.GetUnprocessedMessagesAfterTimestamp(ctw.topicID, milestoneTimestamp)
 	if err != nil {
+		log.Errorf("Could not get unprocessed messages after timestamp [%s]", milestoneTimestamp)
 		log.Fatal(err)
 	}
 	ctw.started = true
+	log.Println(unprocessedMessages)
 	for _, u := range unprocessedMessages.Messages {
-		// validate and check body
-		msg := &protomsg.TopicSignatureMessage{}
+		decodedMessage, err := b64.StdEncoding.DecodeString(u.Message)
+		if err != nil {
+			log.Errorf("Could not decode message - [%s]", u.Message)
+			log.Fatal(err)
+		}
+
+		msg := &validatorproto.TopicSignatureMessage{}
+		err = proto.Unmarshal(decodedMessage, msg)
+		if err != nil {
+			log.Errorf("Could not unmarshal message - [%s]", u.Message)
+			log.Fatal(err)
```

Comment:
It is not a good idea to fatal protobuf unmarshalling - if message of another type is submitted, it is still valid, just not for this scenario.